hazardous/data/_seer.py

```diff
@@ -0,0 +1,213 @@
+"""Helper to reproduce the SEER dataset transformations as performed in SurvTRACE.
+
+See: https://github.com/RyanWangZf/SurvTRACE/blob/main/data/process_seer.py
+"""
+import pandas as pd
+
+CATEGORICAL_COLUMN_NAMES = [
+    "Sex",
+    "Year of diagnosis",
+    "Race recode (W, B, AI, API)",
+    "Histologic Type ICD-O-3",
+    "Laterality",
+    "Sequence number",
+    "ER Status Recode Breast Cancer (1990+)",
+    "PR Status Recode Breast Cancer (1990+)",
+    "Summary stage 2000 (1998-2017)",
+    "RX Summ--Surg Prim Site (1998+)",
+    "Reason no cancer-directed surgery",
+    "First malignant primary indicator",
+    "Diagnostic Confirmation",
+    "Median household income inflation adj to 2019",
+]
+
+NUMERIC_COLUMN_NAMES = [
+    "Regional nodes examined (1988+)",
+    "CS tumor size (2004-2015)",
+    "Total number of benign/borderline tumors for patient",
+    "Total number of in situ/malignant tumors for patient",
+]
+
+# "xxx" names are placeholders, these columns are not used.
+INDICES_TO_COLUMN_NAMES = {
+    0: "Patient ID",
+    1: "Sex",
+    2: "Year of diagnosis",
+    3: "Site recode ICD-0-3/WHO 2008",
+    4: "COD to site recode",
+    5: "Race recode (W, B, AI, API)",
+    6: "Histologic Type ICD-O-3",
+    7: "ICD-O-3 Hist/behav, malignant",
+    8: "Laterality",
+    9: "Sequence number",
+    10: "Vital status recode (study cutoff used)",
+    11: "ER Status Recode Breast Cancer (1990+)",
+    12: "PR Status Recode Breast Cancer (1990+)",
+    13: "Regional nodes examined (1988+)",
+    14: "xxx",
+    15: "Summary stage 2000 (1998-2017)",
+    16: "xxx",
+    17: "Reason no cancer-directed surgery",
+    18: "CS tumor size (2004-2015)",
+    19: "First malignant primary indicator",
+    20: "Diagnostic Confirmation",
+    21: "Total number of benign/borderline tumors for patient",
+    22: "Total number of in situ/malignant tumors for patient",
+    23: "Median household income inflation adj to 2019",
+    24: "RX Summ--Surg Prim Site (1998+)",
+    25: "SEER other cause of death classification",
+    26: "SEER cause-specific death classification",
+    27: "Survival months",
+    28: "xxx",
+}
+
+
+COLUMN_NAMES_TO_DTYPE = {
+    "Histologic Type ICD-O-3": "int64",
+}
+
+
+def load_seer(input_path, rename=True):
```

Comment:
```suggestion
def load_seer(input_path, rename=True, survtrace_preprocessing=False):
```

Let's make it possible to enable or disable the survtrace preprocessing. When the survtrace feature preprocessing is disable, this would just get the raw values (with informative column names and meaningful dtypes.

Reply:
relative to the hazardous source folder, or in the home dir. If the file does not exist, we could raise a ValueError with a link to the instructions to get the raw text file dataset.

Reply:
So, would you look for a file with a specific name in the home dir, or do you aim at replacing the `FileNotFoundError` with more context? 
WDYT about adding a small guide in the documentation and mentioning it here?